doc/code_examples/Tutorial_FileIO.cpp

```diff
@@ -41,16 +40,13 @@ int main(int argc, const char** argv)
   if (argc < 2) return 1;
   // the path to the data should be given on the command line
   String tutorial_data_path(argv[1]);
-  
-  MzXMLFile mzxml;
-  MzMLFile mzml;
 
   // temporary data storage
   PeakMap map;
 
   // convert MzXML to MzML
-  mzxml.load(tutorial_data_path + "/data/Tutorial_FileIO.mzXML", map);
-  mzml.store("Tutorial_FileIO.mzML", map);
+  FileHandler().loadExperiment(tutorial_data_path + "/data/Tutorial_FileIO.mzXML", map, {FileTypes::MZXML});
+  FileHandler().storeExperiment("Tutorial_FileIO.mzML", map, {FileTypes::MZML});
```

Comment:
Tutorial_FileIO would probably be a good start to also add other usage patterns.